packages/quantic/force-app/main/default/lwc/quanticResultLabel/quanticResultLabel.js

```diff
@@ -172,7 +172,7 @@ export default class QuanticResultLabel extends LightningElement {
       case 'knowledge':
```

Comment:
Should we add all known Salesforce object types in this `switch` (e.g., Account, Case, Contact, Opportunity, etc)? Or maybe we'll do that when tackling the "Service" use cases?

Instead of mapping all object types by hand in this `switch/case`, what would you think of using a convention to infer the label key by using the objecttype field value? I think it would scale better.

Reply:
> ...considering labels need to be explicitly imported

That is a good point indeed. I suppose there is a way to do that with LWC, but I don't know how. Let's keep the code as is for the moment.